pandas/tests/io/test_excel.py

```diff
@@ -1440,6 +1440,62 @@ def test_excel_date_datetime_format(self):
                 # to use df_expected to check the result
                 tm.assert_frame_equal(rs2, df_expected)
 
+    # GH19242 - test writing Interval without labels
+    def test_to_excel_interval_no_label(self):
+        _skip_if_no_xlrd()
+
+        with ensure_clean(self.ext) as path:
+            rand = np.random.randint(-10, 10, size=(20, 2))
+            frame = DataFrame(rand)
+            intervals = pd.cut(frame[0], 10)
+            frame['new'] = intervals
```

Comment:
can you make this a little less verbose
use expected for the comparison variable